elasticsearch/README.md

```diff
@@ -49,4 +49,40 @@ e.g. [for `dev-merit`](https://grafana.dev.merit.uw.systems/d/4yyL6dBMk/elastics
 
 ## Backup
 
-TODO
+ElasticSearch comes with an option to backup the data to S3 bucket.
+In order to enable it, you need to:
+
+- have an S3 bucket together with user that has access to it,
+- provide the bucket user credentials to ElasticSearch keystore, and
+- configure the backup via API or with the Kibana Stack Management UI.
+
+### Steps
+
+1. Create bucket and user via Terraform
+   Example bucket and user configuration can be
+   found [here](https://github.com/utilitywarehouse/terraform/blob/master/aws/dev/dev-enablement/test-backups-s3-bucket.tf).
+- both bucket and user can be configured with (
+  system-terraform-modules)[https://github.com/utilitywarehouse/system-terraform-modules]
+- sadly, we must use AWS user instead of role, which means that we must deal with static set of credentials instead of
+  rolling them automatically with Vault. This is due to the fact that ElasticSearch supports either static credentials
+  or IAM role token created by OIDC (available only in EKS)
+  ([source](https://www.elastic.co/guide/en/elasticsearch/reference/8.11/repository-s3.html#iam-kubernetes-service-accounts)).
+  Once you have run `make plan` and `make apply` you can capture the outputs you need using command
+  `make show ARGS="-json"`.
+
+2. Patch the ElasticSearch manifest in order to provide the credentials
+   Example patch can be found [here](example/env-patch.yaml).
```

Comment:
Maybe we could link to our patch in kubernetes manifests (could have both links)? Since a real-world example is more likely to not be incorrect/out of date

Reply:
[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/35/commits/7123350dcb716b5e7abf703a494c0f6a632c4509#diff-bdc6df8fb3c01e36aa0a2781c9487c4157a9fb13902116f5e8d6e2c3724b59dfR74)